### PR TITLE
Updated crypto to support new encryption scheme

### PR DESCRIPTION
**Pull request by angelRev**

### CODE REVIEW

File: webex_assistant_sdk/crypto.py [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
+
+def get_file_contents(filename: str) -> bytes:
+    with open(filename, 'rb') as f:
+        data = f.read()
+    return data
+
 
-    return True
+def load_private_key_from_file(filename: str):
+    key_data = get_file_contents(filename)
```
Comment (snow0x2d0) [inline webex_assistant_sdk/crypto.py:53]: You can actually just use `pathlib.Path` here: `Path(filename).read_bytes()`

**Review by snow0x2d0 — CHANGES_REQUESTED** [review]
One minor comment but once the tests pass this looks good to me.

File: webex_assistant_sdk/crypto.py [hunk 2]
```diff
@@ -1,46 +1,106 @@
...
-        h.verify(bytes.fromhex(signature))
-    except InvalidSignature:
-        return False
+        private_key: RSAPrivateKey = load_ssh_private_key(private_key_bytes, None)
+        return private_key
+    except (binascii.Error, ValueError, UnsupportedAlgorithm) as ex:
+        raise EncryptionKeyError('Unable to load private key') from ex
+
+
+def load_private_key_from_file(filename: str):
```
Comment (snow0x2d0) [inline webex_assistant_sdk/crypto.py:48]: Missing a return type here

File: webex_assistant_sdk/crypto.py [hunk 3]
```diff
@@ -1,46 +1,106 @@
...
+    key_data = Path(filename).read_bytes()
+    private_key = load_private_key(key_data)
+    return private_key
+
+
+def load_public_key_from_file(filename: str) -> str:
+    key_data = Path(filename).read_bytes()
+    return key_data.decode('utf-8')
```
Comment (snow0x2d0) [inline webex_assistant_sdk/crypto.py:56]: I feel like it makes more sense if the load public/private key methods return the same types, either both return the bytes or both return the corresponding cryptography object.

**Review by snow0x2d0 — CHANGES_REQUESTED** [review]
Two small things but looks great overall 👍

File: webex_assistant_sdk/crypto.py (hunk 2 above)
Comment (angelRev) [reply webex_assistant_sdk/crypto.py]: Done

File: webex_assistant_sdk/crypto.py (hunk 3 above)
Comment (angelRev) [reply webex_assistant_sdk/crypto.py:57]: Done

**Review by snow0x2d0 — APPROVED** [review]

**Pull request merged**